src/box/memtx_engine.cc

```diff
@@ -1369,6 +1370,10 @@ memtx_engine_new(const char *snap_dirname, bool force_recovery,
 	if (memtx->gc_fiber == NULL)
 		goto fail;
 
+	/* Small memtx_memory makes the initialization fail. */
```

Comment:
And there the comment must completely describe why the limit is needed and where it came from.